### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by JeremyMordkoff** [telekom/das-schiff#13] [General] [answered]
It seems that it has been many months since the last update. Is this project active?

**Comment by Cellebyte** [comment]
@JeremyMordkoff that is true at least in this project we are not actively updating.

we currently work on OpenSourcing parts of our work in the next months so keep an eye on the organization.

**Comment by schrej** [comment, marked as answer]
Hey @JeremyMordkoff, sorry that it took this long, but we've just updated the repo with our latest architecture. We've also open sourced two new projects which are linked in the readme.
